### PR TITLE
eeg doc clean

### PR DESCRIPTION
**Pull request by Lx37**
Signed-off-by: Lx37 alexandra.corneyllie@gmail.com

### CODE REVIEW

File: pyacq/devices/brainampsocket.py [hunk 1]
```diff
@@ -98,6 +98,7 @@ class BrainAmpSocket(Node):
     
     This class is a bridge between pyacq and the socket-based data streaming
     provided by the Vision recorder acquisition software.
+    (configuration)
```
Comment (samuelgarcia) [inline pyacq/devices/brainampsocket.py]: ??

File: pyacq/devices/eeg_emotiv.py [hunk 2]
```diff
@@ -140,9 +137,16 @@ def __init__(self, **kargs):
         self.device_info = dict()
 
     def _configure(self, device_handle='/dev/hidraw0'):
-        
+        '''
+        Parameters
+        ----------
+        device_handle : str for Linux, hid object for windows
```
Comment (samuelgarcia) [inline pyacq/devices/eeg_emotiv.py]: I think this is the device_path and not the hid object it self.

File: pyacq/devices/eeg_emotiv.py [hunk 3]
```diff
@@ -140,9 +137,14 @@ def __init__(self, **kargs):
         self.device_info = dict()
 
     def _configure(self, device_handle='/dev/hidraw0'):
-        
+        '''
+        Parameters
+        ----------
+        device_handle : str
+            Path to the usb hidraw used
```
Comment (campagnola) [inline pyacq/devices/eeg_emotiv.py]: Would be helpful to have something like `(for example, '/dev/hidraw0' on linux or '.....' on windows)`

**Comment by Lx37** [comment]
This modificationss are added to the specific branch of each device

**Pull request closed without merging**